shell/platform/common/accessibility_bridge_unittests.cc

```diff
@@ -14,45 +14,40 @@ namespace testing {
 
 using ::testing::Contains;
 
+FlutterSemanticsNode CreateSemanticsNode(
+    int32_t id,
+    const char* label,
+    const std::vector<int32_t>* children = nullptr) {
+  return {
+      .id = id,
+      .flags = static_cast<FlutterSemanticsFlag>(0),
+      .actions = static_cast<FlutterSemanticsAction>(0),
+      .text_selection_base = -1,
+      .text_selection_extent = -1,
+      .label = label,
+      .hint = "",
+      .value = "",
+      .increased_value = "",
+      .decreased_value = "",
+      .child_count = children ? children->size() : 0,
+      .children_in_traversal_order = children ? children->data() : nullptr,
+      .custom_accessibility_actions_count = 0,
+  };
+}
```

Comment:
The tests below are the same as before but now use `CreateSemanticsNode` to reduce noise.

Reply:
Thanks for cleaning up!